cpp/FbxCliWrapper/FbxCliWriter.cpp

```diff
@@ -0,0 +1,188 @@
+#include "FbxCliWriter.h"
+#include <assert.h>
+
+
+namespace FbxClrWrapper

```

Comment:
Wait, what kind of abomination is this?? Six-space tabs? My eyes ... they burn. ;-)

Reply:
I'm not seeing any 6 space tabs... Where exactly?  I only see double white line...

Reply:
![image](https://user-images.githubusercontent.com/1759994/59854710-6baa7000-9341-11e9-9519-8e174fc766e8.png)

Reply:
Actually it looks like 8 spaces?

Reply:
As far as I can tell, that is github showing you the super long tabs - there is one tab character and it's putting 8 spaces...